services/consensusalgo/benchmarkconsensus/leader.go

```diff
@@ -26,6 +26,8 @@ func (s *service) leaderConsensusRoundRunLoop(ctx context.Context) {
 
 	s.lastCommittedBlock = s.leaderGenerateGenesisBlock()
 	for {
+		meter := s.logger.Meter("consensus-round-tick")
```

Comment:
here as well, a better approach would be to measure distinct branches of behavior: one for successful round, one for timeout and one for error. The API for `Meter` might need to change - can you think of a better approach?

Reply:
I think it makes sense, we should rethink the API.